src/controllers/contactController.js

```diff
@@ -0,0 +1,27 @@
+import ContactModel from "../models/contact";
+
+export function list(req, res) {
+    ContactModel.find({})
+    .exec()
+    .then((contacts) => {
+        return res.json(contacts);
+    });
+}
+
+export function show(req, res) {
+    ContactModel.findById(req.params.id)
+    .exec()
+    .then((contact) => {
+        return res.json(contact);
+    })
+}
+
+export function create(req, res) {
+    const contact = new ContactModel({
+        body: req.body.body
```

Comment:
Looks like we forgot to use the properties relevant to a contact
I think you did the same for Product, Contact and Vehicle